### ISSUE TITLE
[Installer] Uncaught unknown database error

### ISSUE THREAD

**Issue by rolebi** [Sylius/Sylius#2454]
When  running installer I get the following error :

```
Step 2 of 4. Setting up the database.
Creating Sylius database for environment dev.
PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[42000] [1049] Unknown database 'xxxx'' in vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43
```
The `InstallDatabaseCommand::isDatabasePresent` does not match the given exception message.

**Comment by kayue** [comment]
Can be close now I think.

**Comment by rolebi** [comment]
Fixed by #2453

**Comment by pjedrzejewski** [comment]
:+1:
